packages/react-liveness/src/components/FaceLivenessDetector/service/utils/__tests__/liveness.test.ts

```diff
@@ -54,10 +54,44 @@ describe('Liveness Helper', () => {
           sessionInfo
         );
 
-      expect(faceMatchState).toBe(FaceMatchState.TOO_CLOSE);
+      expect(faceMatchState).toBe(FaceMatchState.OFF_CENTER);
       expect(faceMatchPercentage).toBe(0);
     });
 
+    it('should return TOO FAR', () => {
+      const face: Face = {
+        height: 586,
+        width: 586,
+        left: -6,
+        top: 21,
+        timestampMs: 1,
+        rightEye: [185, 231],
+        leftEye: [427, 224],
+        mouth: [330, 493],
+        nose: [334, 400],
+        rightEar: [36, 243],
+        leftEar: [518, 217],
+      };
+      const ovalDetails: LivenessOvalDetails = {
+        height: 465,
+        width: 288,
+        flippedCenterX: 320,
+        centerX: 320,
+        centerY: 240,
+      };
+      const initialIntersection: number = 0.3;
+      const sessionInfo = mockSessionInformation;
+
+      const { faceMatchState } = getFaceMatchStateInLivenessOval(
+        face,
+        ovalDetails,
+        initialIntersection,
+        sessionInfo
+      );
+
+      expect(faceMatchState).toBe(FaceMatchState.TOO_CLOSE);
```

Comment:
Test reads it should return Too Far but expecting too close here?

Reply:
Will fix, copy paste mistake